src/vaadin-grid-sort-mixin.js

```diff
@@ -51,31 +51,63 @@ export const SortMixin = (superClass) =>
     /** @private */
     _onSorterChanged(e) {
       const sorter = e.target;
+      e.stopPropagation();
+      this.__updateSorter(sorter);
+      this.__applySorters();
+    }
+
+    /** @private */
+    __removeSorters(sorters) {
+      if (sorters.length == 0) {
+        return;
+      }
+
+      sorters.forEach((sorter) => {
+        this._removeArrayItem(this._sorters, sorter);
```

Comment:
This uses a different method of removing items from an array than what is used above in `__removeFilters`, I would suggest to align them.

Reply:
Done